packages/lesswrong/components/ea-forum/EAHome.tsx

```diff
@@ -1,23 +1,28 @@
 import React from 'react'
-import { userHasEAHomeHandbook } from '../../lib/betas'
-import { PublicInstanceSetting } from '../../lib/instanceSettings'
+// import { userHasEAHomeHandbook } from '../../lib/betas'
+// import { PublicInstanceSetting } from '../../lib/instanceSettings'
+import { DatabasePublicSetting } from '../../lib/publicSettings'
 import { Components, registerComponent } from '../../lib/vulcan-lib'
 import { useCurrentUser } from '../common/withUser'
 
-const eaHomeSequenceIdSetting = new PublicInstanceSetting<string | null>('eaHomeSequenceId', null, "optional") // Sequence ID for the EAHomeHandbook sequence
+// const eaHomeSequenceIdSetting = new PublicInstanceSetting<string | null>('eaHomeSequenceId', null, "optional") // Sequence ID for the EAHomeHandbook sequence
+const showSmallpoxSetting = new DatabasePublicSetting<boolean>('showSmallpox', false)
 
 const EAHome = () => {
   const currentUser = useCurrentUser();
   const {
-    RecentDiscussionFeed, HomeLatestPosts, EAHomeHandbook, RecommendationsAndCurated
+    RecentDiscussionFeed, HomeLatestPosts, EAHomeHandbook, RecommendationsAndCurated, SmallpoxBanner
   } = Components
 
   const recentDiscussionCommentsPerPost = (currentUser && currentUser.isAdmin) ? 4 : 3;
-  const shouldRenderEAHomeHandbook = userHasEAHomeHandbook(currentUser)
+  // const shouldRenderEAHomeHandbook = userHasEAHomeHandbook(currentUser)
```

Comment:
OK, I think I get it – you're removing the handbook links for now, and then you'll put this back later. TBH I'm not a huge fan of this, it seems brittle, and it raises the question of what's going to happen if you want to add a different banner at some point in the future. I think I'd prefer this to be guarded, either with a db setting like `showHomeHandbookBanner`, or just a check to see if any other banners are going to be rendered. Then the code is more reusable in future – you can easily update the DB setting to get the banner back if you want to run it again next year, and you can use the same pattern for other banners in the future.